randovania/gui/multiplayer_session_window.py

```diff
@@ -403,18 +405,31 @@ async def duplicate_session(self):
             await self._admin_global_action(SessionAdminGlobalAction.DUPLICATE_SESSION, new_name)
 
     async def _check_dangerous_presets(self, permalink: Permalink) -> bool:
+        def _combine(arr: list[list[str]]):
+            return "\n".join(
+                f"{world.name}: {', '.join(dangerous)}"
+                for world, dangerous in zip(self._session.worlds, arr, strict=True)
+                if dangerous
+            )
+
+        all_incompatible_settings = [
+            preset.settings_incompatible_with_multiworld()
+            for preset in permalink.parameters.presets
+        ]
+        if any(all_incompatible_settings):
+            message = ("The following presets have settings that are incompatible with Multiworld:\n"
```

Comment:
strictly speaking, this isn't displaying the *presets* any more, but the world using that preset, right?

Reply:
oh yeah